qiskit_experiments/library/tomography/fitters/scipy_lstsq.py

```diff
@@ -103,16 +99,27 @@ def scipy_linear_lstsq(
         The fitted matrix rho that maximizes the least-squares likelihood function.
     """
     t_start = time.time()
+
+    if kwargs.pop("conditional_measurement_indices", None):
```

Comment:
Why it is dropped? It was supported by the first implementation. Because of performance?

Reply:
This fitter is being removed as soon as this release goes out, so I didn't want to go to the effort of adding support for it.